components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/listener/ApplicationTenantMgtListener.java

```diff
@@ -0,0 +1,72 @@
+package org.wso2.carbon.identity.application.mgt.listener;
+
+import org.wso2.carbon.identity.application.common.IdentityApplicationManagementException;
+import org.wso2.carbon.identity.application.mgt.ApplicationManagementService;
+import org.wso2.carbon.stratos.common.beans.TenantInfoBean;
+import org.wso2.carbon.stratos.common.exception.StratosException;
+import org.wso2.carbon.stratos.common.listeners.TenantMgtListener;
+
+/**
+ * Application Tenant Management Listener
+ */
+public class ApplicationTenantMgtListener implements TenantMgtListener {
```

Comment:
Use org.wso2.carbon.identity.oauth2.listener.TenantCreationEventListener

Reply:
Removed `ApplicationTenantMgtListener` and will use `org.wso2.carbon.identity.oauth2.listener.TenantCreationEventListener` instead.